packages/documentation/src/stories/components/datepicker/datepicker.stories.ts

```diff
@@ -0,0 +1,207 @@
+import { Args, StoryObj, StoryContext, StoryFn } from '@storybook/web-components-vite';
+import { html } from 'lit';
+import { MetaComponent } from '@root/types';
+
+const meta: MetaComponent = {
+  id: 'eb77cd02-48b2-42e1-a3e4-cd8a973d431e',
+  title: 'Components/Datepicker',
+  component: 'post-datepicker',
+  tags: ['package:WebComponents'],
+  render,
+  exclude: ['id'],
+  parameters: {
+    badges: [],
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/JIT5AdGYqv6bDRpfBPV8XR/Foundations---Components-Next-Level?node-id=21044-101376&t=FNgdl36MZPMbc1Bs-4',
+    },
+  },
+  args: {
+    id: 'main',
```

Comment:
Is this argument necessary in production?
Because for the DEVs the more important attribute will be the `name` or `id` attribute on the input fields themself.